Cogs/verification.py

```diff
@@ -18,52 +18,33 @@ class Verification(commands.Cog):
     def __init__(self, bot):
         self.bot=bot
 ###########################################################################
-    @commands.command(name='verify')
-    async def verify(self, ctx, email):
+    @discord.slash_command(name='verify', description="Verifies a user for access to locked channels")
+    async def verify(self, ctx, email, firstname, lastname):
         if(discord.utils.get(ctx.author.roles, name = "Verified") != None): # We could have this check the DB, would maybe cause issues with manually verified folks.
-            await ctx.send(embed = discord.Embed(title = "You are already verified!", description = "*If you believe this is an error, please message a board member*", color = discord.Color.red()))
-            await ctx.message.delete()
+            await ctx.respond(embed = discord.Embed(title = "You are already verified!", description = "*If you believe this is an error, please message a board member*", color = discord.Color.red()))
             return
-
-        #Check if email has starting/ending parenthesis
-        if (email[0] == "("):
-            email = email[1:]
-        if (email[-1] == ")"):
-            email = email[:-1]
+        

```

Comment:
I would add an await ctx.defer() here just in case the email checking takes longer for some reason. Dont want discord to say we didnt respond just cause of that.